### DISCUSSION TITLE
[vite-plugin-remote-assets] socket hang up

### DISCUSSION THREAD

**Discussion by aladdin-add** [slidevjs/slidev#373] [Q&A] [answered]
it might be related to my settings. it's working fine when dev, but got an error when running`slidev build`:

![image](https://user-images.githubusercontent.com/13050025/137287928-fa243602-8e0f-4fed-bc5a-3115b677af56.png)

env:
OS: macOS 12.0
Node: 16.11.1
npm: 8.0.0
@slidev/cli@0.22.7
@slidev/theme-default@0.19.1
@slidev/theme-seriph@0.19.1  

ps: I've tested the url https://vuejs.org/images/icons/favicon-96x96.png can be opened in my browsers.

**Comment by antfu** [comment, marked as answer]
Remote assets have been disabled by default from 0.23. Consider upgrading to the latest version.

**Reply by aladdin-add** [reply]
will try, thx!
